### ISSUE TITLE
[New Sensor]: BlueMaestro (maybe old ones)

### ISSUE THREAD

**Issue by dobestler** [custom-components/ble_monitor#1082]
### Sensor Description

BlueMaestro TempHum

### Additional information

https://bluemaestro.com/products/tempo-disc-bluetooth-temperature-humidity-sensor-beacon-logger (or an older version of it)

My main problem is, that I see the advertisements in the logs but the devices don't show up. AFAIK it's because there is no integration for them?
I think I am doing something wrong since I am a completely new to HA.

### BLE advertisements

```shell
2023-01-03 09:32:09.244 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration ble_monitor which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-01-03 09:32:09.247 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-01-03 09:32:12.734 DEBUG (MainThread) [homeassistant.components.bluetooth.scanner] Initializing bluetooth scanner with {'detection_callback': <bound method HaScanner._async_detection_callback of <homeassistant.components.bluetooth.scanner.HaScanner object at 0x7f933c6ce0>>, 'scanning_mode': 'active', 'adapter': 'hci0'}
2023-01-03 09:32:12.735 DEBUG (MainThread) [homeassistant.components.bluetooth.scanner] hci0 (E4:5F:AA:AA:DE:75): Starting bluetooth discovery attempt: (1/3)
2023-01-03 09:32:12.823 DEBUG (MainThread) [bleak_retry_connector] Restored 14 discoveries for hci0
2023-01-03 09:32:12.823 DEBUG (MainThread) [homeassistant.components.bluetooth.manager] Registering scanner hci0 (E4:5F:AA:AA:DE:75)
2023-01-03 09:32:13.434 DEBUG (MainThread) [homeassistant.components.bluetooth.manager] hci0 (E4:5F:AA:AA:DE:75) [connectable]: F0:E7:XX:XX:1F:D8 AdvertisementData(local_name='TempHum3', manufacturer_data={307: b'\x16Y\x0e\x10\x07\x01\x00\xae\x02\xee\x00|\x01\x00'}, rssi=-88) match: {'bluemaestro'}
2023-01-03 09:32:14.042 DEBUG (MainThread) [homeassistant.components.bluetooth.manager] hci0 (E4:5F:AA:AA:DE:75) [connectable]: DC:6C:ZZ:ZZ:96:97 AdvertisementData(local_name='TempHum3', manufacturer_data={307: b'\x16V\x0e\x10\x04R\x00\xce\x02+\x00u\x01\x00'}, rssi=-88) match: {'bluemaestro'}

...

2023-01-03 09:59:22.656 DEBUG (MainThread) [homeassistant.components.bluetooth.manager] hci0 (E4:5F:AA:AA:DE:75) [connectable]: F0:E7:XX:XX:1F:D8 AdvertisementData(local_name='TempHum3', manufacturer_data={307: b'\x16Y\x0e\x10\x07\x01\x00\xad\x02\xf0\x00|\x01\x00'}, rssi=-89) match: set()
2023-01-03 09:59:23.199 DEBUG (MainThread) [homeassistant.components.bluetooth.manager] hci0 (E4:5F:AA:AA:DE:75) [connectable]: DC:6C:ZZ:ZZ:96:97 AdvertisementData(local_name='TempHum3', manufacturer_data={307: b'\x16V\x0e\x10\x04S\x00\xcf\x02+\x00v\x01\x00'}, rssi=-90) match: set()
```

**Comment by Ernst79** [comment]
It has a different device_id (`0x16`) in stead of `0x17`. Will be fixed in 11.2.1.

**Comment by dobestler** [comment]
Works like a charm :)
